packages/completer/src/handler.ts

```diff
@@ -7,7 +7,12 @@ import { Text } from '@jupyterlab/coreutils';
 
```

Comment:
Could you add a little docstring somewhere, wherever you think is most appropriate, generally talking about how we had this old completion data structures that worked for the kernel, but now we extended them to provide more hooks for the UI, and if you are a new completion engine, you should use the new ones?